mapper/orm/src/main/java/org/hibernate/search/mapper/orm/logging/impl/Log.java

```diff
@@ -294,4 +294,8 @@ SearchException foundMultipleEntitiesForDocumentId(String entityName, String doc
 	@Message(id = ID_OFFSET + 43, value = "Outbox-generated entity mapping: %1$s")
 	void outboxGeneratedEntityMapping(String xmlMappingDefinition);
 
+	@LogMessage(level = INFO)
+	@Message(id = ID_OFFSET + 44, value = "Session factory is closed. Hibernate Search is probably shutting down.")
+	void sessionFactoryIsClosedOnOutboxProcessing();
```

Comment:
This can probably be removed after you address my other comments

Reply:
I don't think we can address the related comment.